src/labels/CustomLabels.labels

```diff
@@ -2504,6 +2504,27 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Header and text for Field Selection</shortDescription>
         <value>Select Fields</value>
     </labels>
+    <labels>
+        <fullName>bgeEditBatchFieldOptionsWarning</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warning message for Field Options step of BGE Config Wizard</shortDescription>
+        <value>WARNING: There are records saved to this batch. Any changes you make won&apos;t affect existing records.</value>
+    </labels>
+    <labels>
+        <fullName>bgeEditBatchFieldSelectionWarning</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warning msg for Select Fields step of BGE Config Wizard</shortDescription>
```

Comment:
msg -> message

Reply:
I was shortening to get under char limit, but in the end this shortening wasn't necessary. Thanks. :)